bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooglobalfunc.py

```diff
@@ -0,0 +1,148 @@
+# Authors:
+# * Jonas Rembser 06/2021
+# * Harshal Shende 06/2021
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+
+r"""
+/**
+\class RooGlobalFunc
+\brief \parblock \endparblock
+\htmlonly
+<div class="pyrootbox">
+\endhtmlonly
+
+## PyROOT
+
+Some member functions of RooGlobalFunc that take a RooCmdArg as argument also support keyword arguments.
+So far, this applies to FitOptions, Format, Frame, MultiArg, YVar and ZVar.
+
+# Directly passing a RooCmdArg:
+ROOT.RooMCStudy(model, ROOT.RooArgSet(x), ROOT.RooFit.FitOptions=(ROOT.RooFit.Save(True), ROOT.RooFit.PrintEvalErrors(0)))
+
+# With keyword arguments:
+ROOT.RooMCStudy(model, ROOT.RooArgSet(x), FitOptions=dict(Save=True, PrintEvalErrors=0))
+
+\htmlonly
+</div>
+\endhtmlonly
+*/
+"""
+
+from ._utils import _kwargs_to_roocmdargs, _string_to_root_attribute
+
+RooFit = None
+
+# Color and Style dictionary to define matplotlib conventions
+_color_map = {
+    "r": "kRed",
+    "b": "kBlue",
+    "g": "kGreen",
+    "y": "kYellow",
+    "w": "kWhite",
+    "k": "kBlack",
+    "m": "kMagenta",
+    "c": "kCyan",
+}
+_linestyle_map = {"-": "kSolid", "--": "kDashed", ":": "kDotted", "-.": "kDashDotted"}
+
+
+def _RooFit():
+    """Does lazy import of RooFit namespace and then returns it.
```

Comment:
I would've just done the import inside each function below and get the same laziness.

I wonder if the extra import cost is anyway negligible - the link you provide here made a test with Python2.3 (!), so a lot of water has gone under the bridge :smile: